app/utils/fields-validations.ts

```diff
@@ -257,25 +261,27 @@ function statementOfTruthValidation(obj: object): null | ValidationErrors {
 }
 
 function addressValidation(obj: object): null | ValidationErrors {
-  const schema = Joi.object({
-    ['address-line-1']: Joi.string().required().messages({ 'string.empty': i18n.validationErrors.address.line1Required }),
-    ['address-town']: Joi.string().required().messages({ 'string.empty': i18n.validationErrors.address.townCityRequired }),
-    ['address-county']: Joi.string().optional().empty(''),
-    ['address-line-2']: Joi.string().optional().empty(''),
-    ['address-postcode']: Joi.string().optional().empty('').regex(postcodeRegex).messages({
+
+  const schema = joi.object({
+    ['address-line-1']: joi.string().required().messages({ 'string.empty': i18n.validationErrors.address.line1Required }),
+    ['address-town']: joi.string().required().messages({ 'string.empty': i18n.validationErrors.address.townCityRequired }),
+    ['address-county']: joi.string().optional().empty(''),
+    ['address-line-2']: joi.string().optional().empty(''),
+    ['address-postcode']: joi.string().regex(postcodeRegex).messages({
+      'string.empty': i18n.validationErrors.address.postcodeRequired,
```

Comment:
This has been removed not think you have merged in master changes incorrectly

Reply:
I'll double-check, Thank you Chris